manim/mobject/three_dimensions.py

```diff
@@ -165,15 +165,19 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
             face.set_fill(colors[c_index], opacity=opacity)
         return self
 
-    def set_fill_by_value(self, axes: "Mobject", colors: Union[Iterable[Color], Color]):
+    def set_fill_by_value(
+        self, axis: str, axes: "Mobject", colors: Union[Iterable[Color], Color]
```

Comment:
Add a default value for axis as the z axis? To make sure users don't need to change a lot of their current code. Maybe also add the parameter as the last parameter.